ocs_ci/utility/utils.py

```diff
@@ -1822,3 +1829,63 @@ def get_ocs_version_from_image(image):
     except ValueError:
         log.error(f"The version: {version} couldn't be parsed!")
         raise
+
+
+def get_available_ocp_versions(channel):
+    """
+    Find all available OCP versions for specific channel.
+
+    Args:
+        channel (str): Channel of OCP (e.g. stable-4.2 or fast-4.2)
+
+    Returns
+        list: Sorted list with OCP versions for specified channel.
+
+    """
+    headers = {'Accept': 'application/json'}
+    req = requests.get(
+        constants.OPENSHIFT_UPGRADE_INFO_API.format(channel=channel),
+        headers=headers
+    )
+    data = req.json()
+    versions = [Version(node['version']) for node in data['nodes']]
+    versions.sort()
+    return versions
+
+
+def get_latest_ocp_version(channel, index=-1):
+    """
+    Find latest OCP version for specific channel.
+
+    Args:
+        channel (str): Channel of OCP (e.g. stable-4.2 or fast-4.2)
+        index (int): Index to get from all available versions (default -1)
```

Comment:
If I understand the functionality here correctly, the `index` parameter slightly breaks/misleads the clear description of the functionality of this function ... in the way, that it doesn't always return _latest ocp version_ ... because it deeply depends on the provided index.